src/clj/game/cards/hardware.clj

```diff
@@ -4,9 +4,20 @@
   {"Acacia"
    {:events {:pre-purge {:effect (req (let [virus (filter #(has-subtype? % "Virus") (all-installed state :runner))
                                             counters (reduce + (map #(get-virus-counters state :runner %) virus))]
```

Comment:
We may need a UFAQ ruling on this interaction with Hivemind down the road. We can leave it for now.